arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -237,32 +237,35 @@ void dropCollectionFromAllViews(
 ) {
   auto* vocbase = db.useDatabase(dbId);
```

Comment:
are your releasing the db later

Reply:
I haven't a clue, I didn't touch that code
I don't see any release in the original code either